src/styles/index.scss

```
@@ -1,3 +1,197 @@
+@import 'mixins';
+@import 'main';
+@import 'stars';
+@import 'variables';
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+
+  &__link {
+    margin-left: 50px;
+
+    @include flex-center;
+  }
+}
+
+.nav {
+  height: 100%;
+  margin-right: 50px;
+  align-items: stretch;
+
+  &__tabs {
+    height: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: stretch;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__tab {
+    display: flex;
+    margin-left: 10px;
+    margin-right: 10px;
+  }
+
+  &__tabs &__tab:first-child {
+    margin-left: 0;
+  }
+
+  &__tabs &__tab:last-child {
+    margin-right: 0;
+  }
```

better to use one-direction margin
```suggestion
    margin-right: 20px;
    
    &:last-child {
      margin-right: 0;
    }
  }
```